AppAuth-JS/built/logger.d.ts

```diff
@@ -0,0 +1,5 @@
+export declare function log(message: string, ...args: any[]): void;
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork